frontend/cypress/support/commands.ts

```diff
@@ -59,31 +61,94 @@ Cypress.Commands.add('login', (provider: string, username: string, password: str
 					username: ${username},
 					auth_strategy: ${auth_strategy}`
       );
-      // Disabling refresh as this can prevent navigation to another page
-      // until the overview page has fully loaded which can be greatly delayed
-      // if the refresh interval is lower and the api requests are slow.
-      cy.visit('/console/overview?duration=60&refresh=0');
 
       if (auth_strategy === 'openshift') {
-        cy.get('.pf-c-form').contains(auth_strategy, { matchCase: false }).click();
-        cy.get('.pf-c-button').contains(provider, { matchCase: false }).click();
-        cy.get('#inputUsername').clear().type(username);
-        cy.get('#inputPassword').clear().type(password);
-        cy.get('button[type="submit"]').click();
-        cy.wait('@authorized').its('response.statusCode').should('eq', 200);
+        if (provider === 'my_htpasswd_provider') {
```

Comment:
Use the env var rather than hardcode the IdP name since it could be different:

```suggestion
        if (provider === config.env.KUBEADMIIN_IDP) {
```

Reply:
:+1: I think it should be `AUTH_PROVIDER` from: https://github.com/kiali/kiali/pull/5270/files#diff-8a2033002a0cacda548d12a931922a40b9e4142b8ce4430a6711c92c550c33ccR5.

Reply:
My suggestion is stupid. I was not paying attention.

Somethign like this is what I think we need:

`if (provider contains 'htpasswd') {`

 that's pseudocode - hopefully you get what I mean

Reply:
Added a separate env var so one can provide a custom name if need be.

Reply:
Removed the separate env var. Now `CYPRESS_AUTH_PROVIDER` other than `ibmcloud` is assumed to be `htpasswd` and the button that is searched for during login will be whatever `CYPRESS_AUTH_PROVIDER` is set to e.g. `my_htpasswd_provider` or `kube:admin`.